jsonld/projects/C01-verified-carbon-standard-projects.json

```
@@ -0,0 +1,50 @@
+{
+  "@context": {
+    "schema": "http://schema.org/",
+    "regen": "http://regen.network/",
+    "qudt": "http://qudt.org/schema/qudt",
+    "unit": "http://qudt.org/2.1/vocab/unit",
+    "xsd": "http://www.w3.org/2001/XMLSchema#"
+  },
+  "@type": "regen:Project",
+  "regen:vcsProjectId": {
+    "@type": "xsd:unsignedInt",
+    "@value": 0
+  },
+  "regen:vcsProjectPage": {
+    "@type": "schema:URL",
+    "@value": ""
+  },
+  "regen:projectProponent": {
+    "@type": "regen:Organization",
+    "schema:name": ""
+  },
+  "regen:vcsProjectType": "",
+  "regen:projectActivity": "",
```

we should provide `schema:name` and `schema:url` here
```suggestion
  "regen:projectActivity": {
   "schema:name": "",
    "schema:url": {
      "@type": "schema:URL",
      "@value": ""
    }
  }
```